sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/CqlDataReaderDAO.java

```diff
@@ -284,6 +287,17 @@ private Record newRecordFromCql(Key key, Iterable<Row> rows) {
         return new RecordImpl(key, compactionIter, changeIter, rawMetadataIter);
     }
 
+    private Record newRecordFromCqlForStash(Key key, Iterable<Row> rows) {
```

Comment:
I don't mind the approach, but this is mostly a duplicate of `newRecordFromCql()`.  What are your thoughts on updating that method to accept a `boolean trackDeltaSizes` parameter and then create `changeIter` either the current way or the way in this method depending on the value?

Reply:
I have no problem with that at all. I took the approach of duplicating just because I knew there was concern about stash times and I was trying to make it as fast as possible.

Reply:
I am good with either implementation. Let me know which you prefer.